ccdproc/log_meta.py

```diff
@@ -98,15 +98,18 @@ def wrapper(*args, **kwd):
             meta_dict = _metadata_to_dict(log_result)
         else:
             # Logging is not turned off, but user did not provide a value
-            # so construct one.
-            key = func.__name__
-            all_args = chain(zip(original_positional_args, args), kwd.items())
-            all_args = ["{0}={1}".format(name,
-                                         _replace_array_with_placeholder(val))
-                        for name, val in all_args]
-            log_val = ", ".join(all_args)
-            log_val = log_val.replace("\n", "")
-            meta_dict = {key: log_val}
+            # so construct one unless the config parameter auto_logging is set to False
+            if ccdproc.conf.auto_logging is True:
```

Comment:
Would you mind changing this to: `if ccdproc.conf.auto_logging:` (no need for the `is True`)?